### PR TITLE
add animation duration as prop

### PR DESCRIPTION
**Pull request by ericdfields**
1s is too slow :) … this makes it just another property

### CODE REVIEW

File: src/circle.tsx [hunk 1]
```diff
@@ -53,12 +55,12 @@ export class Circle extends Component<CircleProps, CircleState> {
 
   render() {
     const { text } = this;
-    const { progress, size, bgColor, progressColor, lineWidth, animate, roundedStroke, responsive } = this.props;
+    const { progress, size, bgColor, progressColor, lineWidth, animate, animationDuration, roundedStroke, responsive } = this.props;
     const strokeDashoffset = getOffset(progress);
-    const transition = animate ? 'stroke-dashoffset 1s ease-out' : null;
+    const transition = animate ? 'stroke-dashoffset ' + animationDuration + ' ease-out' : null;
```
Comment (styfle) [inline src/circle.tsx]: Should this use ES6 templates instead?

```js
const transition = animate ? `stroke-dashoffset ${animationDuration} ease-out` : null;
```

**Comment by ericdfields** [comment]
Definitely. Wasn’t sure if you supported it.

File: src/circle.tsx (hunk 1 above)
Comment (zzarcon) [reply src/circle.tsx]: +1 to this, can you please add it as part of this PR?

**Comment by ericdfields** [comment]
Let me know if that test looks okay. Wrote it before takeoff with not enough coffee :)

**Comment by zzarcon** [comment]
Tests looks good to me, but the build is failing atm... not sure why /:

**Comment by ericdfields** [comment]
I think I see why.

**Comment by vilvadot** [comment]
Check the travis details. Apparently the tests are broken.

**Comment by ericdfields** [comment]
Right. I’d love to test locally but `ts-react-toolbox test` is erroring.

**Comment by ericdfields** [comment]
@vilvadot @zzarcon OK, got the test suite running. Dumb thing on my end. I've confirmed the tests are failing locally.

Please teach me to fish: how do I update the snapshot? `jest --updateSnapshot` doesn't work out of the box for me because the test is written in typescript. I'm missing something.

From `yarn run test`, my options are

```
Watch Usage
 › Press a to run all tests.
 › Press f to run only failed tests.
 › Press p to filter by a filename regex pattern.
 › Press t to filter by a test name regex pattern.
 › Press q to quit watch mode.
 › Press Enter to trigger a test run.
```

**Comment by zzarcon** [comment]
@ericdfields I think you can just run `$ yarn test` latter press `a` to run all tests and whenever the snapshot fails, you can press `u` from the ci, that will update the snapshots to the new version 👍

**Comment by ericdfields** [comment]
All set! Thanks for the help.

**Comment by vilvadot** [comment]
Looks good to me @zzarcon

**Comment by zzarcon** [comment]
Awesome stuff @ericdfields thanks for taking your time and adding the test 🙌

**Pull request merged**